fix phony translation linking error

  | .../usr/lib/crt1.o: In function `_start':
  | .../../sysdeps/i386/elf/start.S:115: undefined reference to `main'
  | collect2: ld returned 1 exit status

Upstream-Status: Pending

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>

diff --git a/translations/translations.pro b/translations/translations.pro
index cdaf04a..24fa668 100644
--- a/translations/translations.pro
+++ b/translations/translations.pro
@@ -20,7 +20,7 @@ updateqm.name = LRELEASE ${QMAKE_FILE_IN}
 updateqm.CONFIG += no_link
 QMAKE_EXTRA_COMPILERS += updateqm
 
-isEmpty(vcproj) {
+!isEmpty(vcproj) {
     QMAKE_LINK = @: IGNORE THIS LINE
     OBJECTS_DIR =
     win32:CONFIG -= embed_manifest_exe
@@ -30,7 +30,7 @@ isEmpty(vcproj) {
     phony_src.input = PHONY_DEPS
     phony_src.output = phony.c
     phony_src.variable_out = GENERATED_SOURCES
-    phony_src.commands = echo int main() { return 0; } > phony.c
+    phony_src.commands = echo \"int main() { return 0; }\" > phony.c
     phony_src.name = CREATE phony.c
     phony_src.CONFIG += combine
     QMAKE_EXTRA_COMPILERS += phony_src
